interpretation/deepseismic_interpretation/azureml_pipelines/README.md

```diff
@@ -0,0 +1,175 @@
+# Integrating with AzureML
+
+## Running a Pipeline in AzureML
+Set the following environment variables:
+```
+BLOB_ACCOUNT_NAME
+BLOB_CONTAINER_NAME
+BLOB_ACCOUNT_KEY
+BLOB_SUB_ID
+AML_COMPUTE_CLUSTER_NAME
+AML_COMPUTE_CLUSTER_MIN_NODES
+AML_COMPUTE_CLUSTER_MAX_NODES
+AML_COMPUTE_CLUSTER_SKU
+```
```

Comment:
![image](https://user-images.githubusercontent.com/8856246/74275743-4e0f8980-4cc9-11ea-96f3-ee2736b26a7b.png)
 
@annazietlow @maxkazmsft

Reply:
OK, sorry about that - I got confused

Reply:
No worries, @maxkazmsft I'll add some lines in our README.md to make sure other users don't get confused here and will resolve this conversation.

Reply:
Continuing this in a separate conversation below. See commit here adding guidance https://github.com/microsoft/seismic-deeplearning/pull/195/commits/279f3367c38369ab21eb418b502b76c6849a4af3 resolving this conversation here.